src/test/java/com/microsoft/sqlserver/jdbc/unit/UTF8SupportTest.java

```diff
@@ -107,7 +107,7 @@ public void testVarchar() throws SQLException {
 
     @BeforeAll
     public static void setUp() throws ClassNotFoundException, SQLException {
-        connection = PrepUtil.getConnection(getConfiguredProperty("mssql_jdbc_test_connection_properties"));
+        connection = PrepUtil.getConnection(AbstractTest.getConnectionString());
```

Comment:
The test already extends `AbstractTest`, `connection = PrepUtil.getConnection(connectionString);` is enough.